### ISSUE TITLE
Add build pipeline

### ISSUE THREAD

**Issue by leif81** [open-dis/open-dis-csharp#10]
This project could use a build pipeline to ensure proposed changes don't break the build. 

For future work we should integrate a unit tests into the build pipeline.

**Comment by mastertnt** [comment]
Hi,

I could if you are interrested, define the build pipeline with nuget package at the final step.

For the unit tests, i can also initiate something.

**Comment by leif81** [comment]
@mastertnt yes that would be very much appreciated! Please submit a pull request when you're ready.

**Comment by mastertnt** [comment]
Ok, i have started.

First, i will submit the build pipeline then think about some tests.

**Comment by leif81** [comment]
Yes the Dis7 folder is for the 2010 version.
 
Yes it's ok to exclude it from the build for now.

**Comment by mastertnt** [comment]
In the second pass, i will include DIS7.
